opacclient/opacapp/src/main/res/values-nl/strings_api_errors.xml

```diff
@@ -0,0 +1,89 @@
+        <item quantity="one">This library only supports one search criterion.</item>
+        <item quantity="other">This library only supports up to %d search criteria.</item>
+    </plurals>
+    <string name="no_criteria_input">You haven\'t input any search criteria.</string>
+    <string name="combination_not_supported">You used a combination of search criteria that is not supported by this library. Please try another search query.</string>
+    <string name="unknown_error_account">Unknown Error. Please check if your login data is correct.</string>
+    <string name="unknown_error_account_with_description">Unknown Error: \"%s\" Please check if your login data is correct.</string>
+    <string name="unknown_error">Unknown error.</string>
+    <string name="unknown_error_with_description">Unknown error: \"%s\"</string>
+    <string name="internal_error">Internal Error.</string>
+    <string name="internal_error_with_description">Internal Error: \"%s\"</string>
+    <string name="login_failed">Login failed.</string>
+    <string name="could_not_load_account">Account could not be loaded.</string>
+    <string name="api_connection_error">Connection error.</string>
+    <string name="lent_until">lent until</string>
+    <string name="subtitle">Subtitle</string>
+    <string name="pica_which_copy">Which copy do you want to reserve/order? (already available copies will be ordered)</string>
+    <string name="no_copy_reservable">There is no reservable copy.</string>
+    <string name="could_not_load_detail">Details could not be loaded. Please try again.</string>
+    <string name="error">Error</string>
+    <string name="download">Download</string>
+    <string name="reminders">warnings</string>
+    <string name="prolonged_abbr">renew.</string>
+    <string name="prolonging_expired">renewal no longer possible</string>
+    <string name="prolonging_impossible">renewal not possible</string>
+    <string name="prolonging_waiting">not yet renewable</string>
+    <string name="wrong_account_data">The user data you entered is wrong.</string>
+    <string name="order">Sort order</string>
+    <string name="order_default">Default order</string>
+    <string name="order_year_asc">By year (descending)</string>
+    <string name="order_category_asc">By category (ascending)</string>
+    <string name="order_year_desc">By year (descending)</string>
+    <string name="order_category_desc">By category (descending)</string>
+    <string name="interlib_branch">Interlibrary: %s</string>
+    <string name="stacks_branch">Stacks: %s</string>
+    <string name="provision_branch">Provision: %s</string>
+    <string name="unsupported_in_library">This feature is not supported in this library.</string>
+    <string name="reservation_warning">This reservation might cost you a fee.</string>
+    <plurals name="reservations_number">
+        <item quantity="one">%d reservation</item>
+        <item quantity="other">%d reservations</item>
+    </plurals>
```

Comment:
I don't think we should add English strings to the Dutch language file :)

Reply:
Oops, will remove those